web/pandas/pdeps/0013-logical-type-system.md

```diff
@@ -0,0 +1,197 @@
+| Series.str.split() | object              | pa.list(pa.string())     | pd.ListDtype(pa.StringDtype()) |
+| Series.dt.date     | object              | pa.date32()              | pd.DateDtype()                 |
+
+The ``Series.dt.date`` example is worth an extra look - with a PDEP-13 logical type system in place we would theoretically have the ability to keep our default ``pd.DatetimeDtype()`` backed by our current NumPy-based array but leverage pyarrow for the ``Series.dt.date`` solution, rather than having to implement a DateArray ourselves.
+
+While this PDEP proposes reusing existing extension types, it also necessitates extending those types with extra metadata:
+
+```python
+class BaseType:
+
+    @property
+    def data_manager -> Literal["numpy", "pyarrow"]:
+        """
+        Who manages the data buffer - NumPy or pyarrow
+        """
+        ...
+
+    @property
+    def physical_type:
+        """
+        For logical types which may have different implementations, what is the
+        actual implementation? For pyarrow strings this may mean pa.string() versus
+        pa.large_string() versrus pa.string_view(); for NumPy this may mean object
+        or their 2.0 string implementation.
+        """
+        ...
+
+    @property
+    def na_marker -> pd.NA|np.nan|pd.NaT:
+        """
+        Sentinel used to denote missing values
+        """
+        ...
+```
+
+``na_marker`` is expected to be read-only (see next section). For advanced users that have a particular need for a storage type, they may be able to construct the data type via ``pd.StringDtype(data_manager=np)`` to assert NumPy managed storage. While the PDEP allows constructing in this fashion, operations against that data make no guarantees that they will respect the storage backend and are free to convert to whichever storage the internals of pandas considers optimal (Arrow will typically be preferred).
+
+### Missing Value Handling
+
+Missing value handling is a tricky area as developers are split between pd.NA semantics versus np.nan, and the transition path from one to the other is not always clear.
+
+Because this PDEP proposes reuse of the existing pandas extension type system, the default missing value marker will consistently be ``pd.NA``. However, to help with backwards compatibility for users that heavily rely on the equality semantics of np.nan, an option of ``pd.na_marker = "legacy"`` can be set. This would mean that the missing value indicator for logical types would be:
+
+| Logical Type      | Default Missing Value | Legacy Missing Value |
+| pd.BooleanDtype() | pd.NA                 | np.nan               |
+| pd.IntXXType()    | pd.NA                 | np.nan               |
+| pd.FloatXXType()  | pd.NA                 | np.nan               |
+| pd.StringDtype()  | pd.NA                 | np.nan               |
+| pd.DatetimeType() | pd.NA                 | pd.NaT               |
```

Comment:
Period, timedelta64

Reply:
Sure I can make this a fully exhaustive list. For the Legacy missing value on `BinaryDtype`, `ListDtype`, `StructDtype` and `MapDtype` do you think `np.nan` or `pd.NA` makes more sense?